common/reviews/api/core.api.md

```diff
@@ -1425,6 +1427,19 @@ function loadImageToCanvas(canvas: HTMLCanvasElement, imageId: string, requestTy
 // @public (undocumented)
 function loadVolume(volumeId: string, options?: VolumeLoaderOptions): Promise<Types.IImageVolume>;
 
+// @public (undocumented)
+type Mat3 = [

```

Comment:
Can't you write this as number[9]?  Also, the Vec library allows defining both the ArrayBuffer format as well as this format using the | notation - that might be worthwhile, as the underlying implementation is faster.